inspirehep/modules/workflows/tasks/manual_merging.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Tasks related to manual merge generic record."""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+from invenio_db import db
+
+from inspire_json_merger.inspire_json_merger import inspire_json_merge
```

Comment:
The API and module structure of `inspire-json-merger` need some love. This should really be
```python
from inspire_json_merger import merge
```
opening an issue against `inspire-json-merger`.

Reply:
I agree, it's in the TODO list :+1: